README.md

```diff
@@ -1,114 +1,103 @@
-# S3 scrubber
+# Neon S3 scrubber
 
-Initial presentation: https://drive.google.com/file/d/16u3qf0A17EReSXYYFxpoFqwuz3nA2wJp/view
+## Usage
 
-A project to go over S3 buckets for storage nodes, checking its contents and removing the data, not present in the console.
```

Comment:
I think it's useful to have some general description here. If this isn't accurate any more, it would still be good to have an accurate short description of what this thing is.

Reply:
6e0059d0be71507817416ead4c31adfe6c835032